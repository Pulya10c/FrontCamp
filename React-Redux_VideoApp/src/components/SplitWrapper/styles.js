import styled from 'styled-components';

export const SplitWrapper = styled.div`
  display: grid;
  grid-gap: 5px;
  grid-template-columns: auto auto;
  justify-content: space-between;
  align-items: center;
  padding: 10px;
`;
