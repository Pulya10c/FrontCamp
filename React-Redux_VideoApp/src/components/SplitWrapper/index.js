import { SplitWrapper } from './styles';

export default SplitWrapper;
