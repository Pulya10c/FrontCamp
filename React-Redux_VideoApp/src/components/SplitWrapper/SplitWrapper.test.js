import React from 'react';
import { shallow } from 'enzyme';
import SplitWrapper from './';

describe('Component SplitWrapper', () => {
  it('should render correctly', () => {
    expect(shallow(<SplitWrapper />)).toMatchSnapshot();
  });
});
