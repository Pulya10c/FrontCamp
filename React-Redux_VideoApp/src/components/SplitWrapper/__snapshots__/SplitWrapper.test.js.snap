// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component SplitWrapper should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  display: grid;
  grid-gap: 5px;
  grid-template-columns: auto auto;
  justify-content: space-between;
  align-items: center;
  padding: 10px;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
